Readme.md

```diff
@@ -0,0 +1,21 @@
+# Darwinia Community Dao Profiles
+This repository serves as a storage for the members' information and NFT images of Darwinia Community Dao. In addition, it provides a streamlined process for contributors to apply and automatically receive their NFTs.
+
+## How to Apply
+To apply for an NFT, follow the steps below:
+
+1. Create an issue in this repository with the application template.
+2. Fill out the application with your specific information.
+3. Wait for a maintainer to review your application and add the approved label.
```

Comment:
how many approves does it need?

Reply:
Issues only require approval from one administrator, but Pull Requests can be configured to require approval from multiple maintainers before they can be merged.